ntoskrnl/include/internal/amd64/ke.h

```diff
@@ -28,6 +28,7 @@
 #define X86_FEATURE_CX8         0x00000100 /* CMPXCHG8B instruction present */
```

Comment:
Looks like these are not even used and all the values are hardcoded in KiGetFeatureBits. Feel free to improve that. Or not :)

Reply:
Simple enough, I think I will do it then. Readablility of code is important.